addons/event/models/event_event.py

```diff
@@ -644,6 +646,13 @@ def mail_attendees(self, template_id, force_send=False, filter_func=lambda self:
             for attendee in event.registration_ids.filtered(filter_func):
                 self.env['mail.template'].browse(template_id).send_mail(attendee.id, force_send=force_send)
 
+    def _get_event_description(self):
+        """ Returns the event description with a link to the event registration page. """
+        max_length = 2000
+        event_url = Markup('<a href="%s/event/%s/register">%s</a>') % (self.get_base_url(), slug(self), self.name)
```

Comment:
I don't really understand why you need Markup here.

More importantly, this URL only becomes available when "website_event" is installed.
So either we remove the URL, either this methods needs to be overridden in website_event somehow.

Reply:
Hello @awa-odoo 
I used markup because I thought we always need to markup the HTML content :sweat_smile: 
https://www.odoo.com/documentation/17.0/developer/reference/backend/security.html#creating-safe-content-using-markup
I will look around point 2.